packages/@stimulus/core/src/application.ts

```diff
@@ -28,13 +29,17 @@ export class Application implements ErrorHandler {
 
   async start() {
     await domReady()
+    this.logDebugActivity("application", "starting")
     this.dispatcher.start()
     this.router.start()
+    this.logDebugActivity("application", "started")
   }
 
   stop() {
+    this.logDebugActivity("application", "stoping")
```

Comment:
```suggestion
    this.logDebugActivity("application", "stopping")
```

source: https://wikidiff.com/stoped/stopped